homeTask_7/homeTask_7/Comparators/AssigneTaskCompare.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace homeTask_7.Comparators
+{
+    class AssigneTaskCompare : IComparer<Employee>
+    {
+        public int Compare([AllowNull] Employee x, [AllowNull] Employee y)
+        {
+            if ((x is IAssigner) & !(y is IAssigner))
+            {
+                return -1;
+            }
+            else if (!(x is IAssigner) & (y is IAssigner))
+            {
+                return 1;
+            }
+            else if ((x is IAssigner) & (y is IAssigner))
+            {
+                return string.Compare(x.Name, y.Name);
```

Comment:
Why you compare name in this case?

Reply:
So, i' ve decided using default **compare** static function from string class, instead of  i**f x.Name[0] > or < y.Name[0]. retorn  f.e **0 1 -1**** and so on. Because string.Compare return needed result for me(0 1 -1)